dashboards/amazon-connect/amazon-connect.yaml

```diff
@@ -15904,250 +19325,425 @@ dashboards:
+                  BarValues:
+                  - CategoricalMeasureField:
+                      AggregationFunction: DISTINCT_COUNT
+                      Column:
+                        ColumnName: contact_id
+                        DataSetIdentifier: resource_connect_view
+                      FieldId: f6d2cc33-0cbb-4c33-84ff-0c80e98fbf21.contact_id.1.1741035404779
                   Category:
                   - DateDimensionField:
                       Column:
-                        ColumnName: contact_end_date_time [contact_summary]
+                        ColumnName: usage_date
                         DataSetIdentifier: resource_connect_view
-                      DateGranularity: HOUR
-                      FieldId: 8b3854b7-95c6-4b5f-803b-5f547fedb897.contact_end_date_time.0.1724179850633
-                      FormatConfiguration:
-                        DateTimeFormat: ddd, MMM D,  HH:mm
-                        NullValueFormatConfiguration:
-                          NullString: 'null'
-                      HierarchyId: 1f26ef3f-0a1f-4353-a158-efa9be7f2b1d
+                      FieldId: f6d2cc33-0cbb-4c33-84ff-0c80e98fbf21.usage_date.0.1741035391889
+                      HierarchyId: f6d2cc33-0cbb-4c33-84ff-0c80e98fbf21.usage_date.0.1741035391889
                   Colors:
                   - CategoricalDimensionField:
                       Column:
-                        ColumnName: __call_type
+                        ColumnName: region
                         DataSetIdentifier: resource_connect_view
-                      FieldId: 7f32a31a-992c-4525-ab22-47e8971ca6b0.2.1724179872682
-                      HierarchyId: c8f06404-c2a5-4744-988a-5cd9189f495c
-                  Values:
-                  - CategoricalMeasureField:
-                      AggregationFunction: DISTINCT_COUNT
+                      FieldId: f6d2cc33-0cbb-4c33-84ff-0c80e98fbf21.country.3.1741035976753
+                      HierarchyId: 1a7e49e2-6b11-4b2a-96a5-d260081afd61
+                  LineValues:
+                  - NumericalMeasureField:
                       Column:
-                        ColumnName: contact_id
+                        ColumnName: SUM Contact Cost
                         DataSetIdentifier: resource_connect_view
-                      FieldId: f6d2cc33-0cbb-4c33-84ff-0c80e98fbf21.contact_id.1.1724179860473
+                      FieldId: 6b8cb866-933f-457b-b956-3f02d9e17500.6.1741037605873
+                      FormatConfiguration:
+                        FormatConfiguration:
+                          CurrencyDisplayFormatConfiguration:
+                            DecimalPlacesConfiguration:
+                              DecimalPlaces: 2
+                            NegativeValueConfiguration:
+                              DisplayMode: POSITIVE
+                            NullValueFormatConfiguration:
+                              NullString: 'null'
+                            Prefix: ''
```

Comment:
Definition.Sheets[5].Visuals[21].ComboChartVisual.ChartConfiguration.FieldWells.ComboChartAggregatedFieldWells.LineValues[0].NumericalMeasureField.FormatConfiguration.FormatConfiguration.CurrencyDisplayFormatConfiguration.Prefix, value: 0, valid min length: 1
Invalid length for parameter